scripts/os_configure/install_software_ros.bash

```diff
@@ -48,12 +64,6 @@ sudo apt -y install ros-${ROS_DISTRO}-moveit
 sudo apt -y install ros-${ROS_DISTRO}-moveit-*
 sudo apt -y install libmuparser-dev ros-${ROS_DISTRO}-brics-actuator ros-${ROS_DISTRO}-openrave ros-${ROS_DISTRO}-move-base
 sudo apt -y install ros-${ROS_DISTRO}-teleop-twist-joy ros-${ROS_DISTRO}-twist-mux
-# Mit dem vorsicht auch: ros-${ROS_DISTRO}-schunk-*
-
-# INSTALL Released repos from KITRobotics
-# sudo apt -y install ros-${ROS_DISTRO}-iirob-filters ros-${ROS_DISTRO}-force-torque-sensor ros-${ROS_DISTRO}-ati-force-torque
-# sudo apt -y install ros-${ROS_DISTRO}-ipr-extern
-
 
 # ROS Control
 sudo apt -y install ros-${ROS_DISTRO}-control-*
```

Comment:
This will not work, we should just install following packaged explicitly:

```suggestion
sudo apt -y install ros-${ROS_DISTRO}-ros2-control ros-${ROS_DISTRO}-ros2-controllers ros-${ROS_DISTRO}-gazebo-ros2-control ros-${ROS_DISTRO}-gz-ros2-control
```

Reply:
Oh, this should be in the ROS2 install file.

Reply:
Done

Reply:
IPR stuff is not present in my file

Reply:
The packages:
- ros-${ROS_DISTRO}-gz-ros2-control 
- ros-${ROS_DISTRO}-joint-state-controller
are note present

Reply:
only ros-${ROS_DISTRO}-gz-ros2-control is not present

Reply:
```suggestion
sudo apt -y install ros-${ROS_DISTRO}-ros2-control ros-${ROS_DISTRO}-ros2-controllers ros-${ROS_DISTRO}-gazebo-ros2-control
```